hw/dv/tools/vcs/unr.cfg

```diff
@@ -0,0 +1,23 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+-covInput ../../../cov_merge/merged.vdb
```

Comment:
Can this file read env vars?

Reply:
Surprisedly, env var works in unr.cfg. This makes things much easier. Thanks.

Reply:
Great! We all have learned something new :-).